### DISCUSSION TITLE
v2.0.0

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense#76] [Announcements]
## Features

- Add `Query` switch to mqtt and webapp setup (defaults to off so as to not break existing fingerprints)
- Add `Active Scan` switch to mqtt and webapp setup
- Add device model to apple fingerprints 
- Add support [room assistant app](https://apps.apple.com/us/app/room-assistant/id1538642237?itsct=apps_box_link&itscg=30200) @DTTerastar (#69)

## Misc Changes

- Stats reworked

<hr /><em>This discussion was created from the release <a href='https://github.com/ESPresense/ESPresense/releases/tag/v2.0.0'>v2.0.0</a>.</em>

**Comment by davidcoulson** [comment]
Will pre-built binaries be available for 1.7.3 and 2.0.0?

**Reply by RutgerDiehard** [reply]
Patiently waiting for all my ESPresense instances to automatically update....

**Reply by RutgerDiehard** [reply]
And just like that, it happened!

**Reply by DTTerastar** [reply]
Yeah, v2.0.0 is up.  I'm just gonna delete v1.7.3

**Comment by DTTerastar** [comment]
Help upvote our issue: https://github.com/h2zero/NimBLE-Arduino/issues/299
